README.md

```diff
@@ -262,16 +264,16 @@ customModules:
         links:
           - title: Overview
             path: /tutorials/9-path-to-prod/1-overview.html
-          - title: Prepare the software
+          - title: Prepare the software to run
             path: /tutorials/9-path-to-prod/2-software.html
           - title: Prepare a validator and keys
             path: /tutorials/9-path-to-prod/3-keys.html
-          - title: Prepare the node start
+          - title: Prepare where the node starts

```

Comment:
EIther `where the node starts` or `where nodes start`. Went with the first option, if that is incorrect, please change accordingly.

Reply:
Mmh, "where nodes start" sounds good too.